site/website/blog/2020-05-20-designing-evergreen-cache-cdc.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Designing an evergreen cache with Change Data Capture
+author: Nicolas Frankel
+authorURL: https://twitter.com/nicolas_frankel
+authorImageURL: https://3l0wd94f0qdd10om8642z9se-wpengine.netdna-ssl.com/wp-content/uploads/2019/12/nicolas-frankel-170x170.jpg
+---
+
+It's been said that there are two things hard in software development, naming things and cache invalidation (while some add off-by-one errors to the mix).
```

Comment:
can you run the markdown linter? you need to wrap your lines among other things `yarn lint:markdow`

Reply:
Yes, I can